### PR TITLE
Iss 269 input antenna height and streaming

### LINKED ISSUE #269: document antenna height
see https://github.com/MerginMaps/input/pull/2522

### PR DESCRIPTION
**Pull request by alex-cit**
fix #269, related to #273 
Added new features that were introduced in Mergin Maps Input 2.1.0:
- antenna height for using  [external GPS](https://merginmaps.com/docs/field/external_gps/)
- new [position variables](https://merginmaps.com/docs/layer/position_variables/)
- [streaming mode ](https://merginmaps.com/docs/field/input_features/) also with distance interval

### CODE REVIEW

File: src/field/input_features.md [hunk 1]
```diff
@@ -6,100 +6,114 @@
...
 
-Snapping can be enabled in your <MainPlatformName /> project to make the field survey easier. [How to Set Up Snapping](../gis/snapping/) will guide you through the snapping options.
+#### Adding points to survey vertices of lines or areas
+Tap **Add** to capture vertices of your line or area. If you want to change the position of the last vertex, tap **Remove**, move the vertex to the correct place and  **Release** it. **Undo** can be used to revert last changes.
+
+Once the survey of the feature is completed, tap **Done** and fill in the form.
+![Capturing lines](./input-capture-line.jpg)
+
+### Streaming mode to survey lines or areas
+Lines and areas can be also captured automatically based on your position. To turn on the streaming mode, long press the **GPS** button. To turn it off, long press the **GPS** button again.
```
Comment (tomasMizera) [inline src/field/input_features.md:44]: A `::: tip/note` might be beneficial here saying that one needs to be in the recording mode. If you hold the GPS button in view mode, nothing happens.

Comment (tomasMizera) [reply src/field/input_features.md:44]: I understand that it is mentioned earlier, it is just that people might google/search for streaming mode and end up here without reading the previous text

File: src/field/input_features.md [hunk 2]
```diff
@@ -6,100 +6,114 @@
...
+![GPS streaming mode to survey lines](./input-streaming-line.jpg "GPS streaming mode to survey lines")
 
-If snapping is enabled, the crosshairs will turn purple and snap to vertices (left) or segments (right) of existing features when capturing new features.
-![input snapping](../gis/snapping/input_basic_snapping.png)
+Once you are finished, tap **Done**. If you are capturing an area, the shape will be automatically closed by connecting the last and the first vertex.
+
+You can change how often you want to capture a vertex in **Line rec. interval** in [**Settings**](./input_ui/#streaming-mode-and-recording-settings). The interval can be defined as time elapsed in seconds or as distance travelled in metres.
+
+:::tip
+<Badge text="since Input 2.1.0" type="tip"/>
```
Comment (tomasMizera) [inline src/field/input_features.md:53]:
```suggestion
<Badge text="since Mergin Maps Input 2.1.0" type="tip"/>
```

**Review by tomasMizera — APPROVED** [review]
Looks good to me, good job! :) 
- image [src/field/input-edit-feature.jpg](https://github.com/MerginMaps/docs/pull/283/files#diff-6a320e18fa8253b991720e2c5720d1479d2e149a4ea9f20e019158eae2a2f82d) looks broken - sync button is out of place,  how did you manage to get it there? 🙂 

I left a couple of minor notes, but I'd say let's release this and fix it later

**Comment by alex-cit** [comment]
>     * image [src/field/input-edit-feature.jpg](https://github.com/MerginMaps/docs/pull/283/files#diff-6a320e18fa8253b991720e2c5720d1479d2e149a4ea9f20e019158eae2a2f82d) looks broken - sync button is out of place,  how did you manage to get it there? slightly_smiling_face

Honestly I don't know, this is an older image (I guess it was created in Input 1.6.0 when the geometry splitting was introduced. So whatever it was, it's probably fixed now ;) Thanks for noticing that, I'll update it!

**Pull request merged**